front/src/ctfnote/tasks.ts

```diff
@@ -59,20 +59,21 @@ export function useSolveTaskPopup() {
   const updateTask = useUpdateTask();
   return (task: Task) => {
     Dialog.create({
-      title: 'Flag:',
+      title: 'Submit flag for ' + task.title,
       color: 'primary',
-      cancel: {
-        label: 'cancel',
-        color: 'warning',
-        flat: true,
-      },
+      class: 'compact-dialog',
       prompt: {
         model: task.flag ?? '',
         type: 'text',
+        filled: true,
+      },
+      cancel: {
+        label: 'Cancel',
+        flat: true,
       },
       ok: {
         color: 'positive',
-        label: 'save',
+        label: 'Save',
       },

```

Comment:
Maybe a hint for this input field is nice? It makes is more consistent with all the other input fields. Old design did not have that either so this is not a regression but just an idea.